driver/examples/src/test/resources/ideprobe.conf

```diff
@@ -0,0 +1,15 @@
+probe {
+
+  driver {
+    headless = false
+    launch.timeout = "600 seconds"
```

Comment:
headless.false is default, launch timeout also should be ok with default. Generally this config is quite messy